lib/MusicBrainz/Server/Edit/Relationship/RelatedEntities.pm

```diff
@@ -25,21 +25,35 @@ around _build_related_entities => sub {
                 map { $_->artist_credit->all_names }
                     @entities;
         }
+    }
+
+    my $link_type =
+        # Remove relationship
+        $self->data->{relationship}{link}{link_type} //
+        # Edit relationship
+        $self->data->{link}{link_type} //
+        # Add relationship
+        $self->data->{link_type};
```

Comment:
This is probably fine since Perl should autovivify the keys if they don't exist, but it'd be slightly cleaner to add `sub link_type { shift->data->{relationship}{link}{link_type} }`, `sub link_type { shift->data->{link}{link_type} }`,  etc. to each relevant package and then do `my $link_type = $self->link_type;` here.